### PR TITLE
update README

### PR DESCRIPTION
**Pull request by ea-open-source**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,63 @@
 # MCP Get Community Servers
 
-TODO: Add description
+A [Model Context Protocol](https://www.anthropic.com/news/model-context-protocol) server for LLMs to interact with Rememberizer Vector Store. See more at: [docs.rememberizer.ai](https://docs.rememberizer.ai/)
```
Comment (oyster) [inline README.md:3]: Let's add a direct link to https://docs.rememberizer.ai/developer/vector-stores.

File: README.md [hunk 2]
```diff
@@ -1,16 +1,63 @@
...
 
 ## Components
 
 ### Resources
 
-The server provides access to two types of resources: Documents or Slack discussions
+The server provides access to your Vector Store's documents.
```
Comment (oyster) [inline README.md:9]: in Rememberizer

File: README.md [hunk 3]
```diff
@@ -1,16 +1,63 @@
...
 
-TODO: Add tools
+1. `rememberizer_vectordb_search`
+
+   - Search for documents in your Vector Store by semantic similarity
+   - Input:
+     - `q` (string): Up to a 400-word sentence to find semantically similar chunks of knowledge
+     - `n` (integer, optional): Number of similar documents to return (default: 5)
+
+2. `rememberizer_vectordb_agentic_search`
```
Comment (oyster) [inline README.md:20]: We need more document for this at https://docs.rememberizer.ai/developer/api-documentations/vector-store/search-for-vector-store-documents-by-semantic-similarity.

File: README.md [hunk 4]
```diff
@@ -1,16 +1,63 @@
...
+     - `document_id` (integer): The ID of the document you want to delete
+
+7. `rememberizer_vectordb_modify_document`
+
+   - Change the name of your Vector Store document
+   - Input:
+     - `document_id` (integer): The ID of the document you want to modify
+
 
 ## Installation
```
Comment (oyster) [inline README.md:62]: We need to show them how to get a Rememberizer VectorStore API key/token.

File: pyproject.toml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 [project]
 name = "mcp-rememberizer-vectordb"
 version = "0.1.1"
-description = "A Model Context Protocol server for interacting with Rememberizer's document and knowledge management API. This server enables Large Language Models to search, retrieve, and manage documents and integrations through Rememberizer."
+description = "A Model Context Protocol server for interacting with Rememberizer Vector Store."
```
Comment (oyster) [inline pyproject.toml:4]: Put a link here to Rememberizer Vector Store here. That link should be used for our PR to their community servers too.

Comment (ea-open-source) [reply pyproject.toml:4]: You mean this link https://docs.rememberizer.ai/developer/vector-stores right anh?

File: README.md (hunk 4 above)
Comment (ea-open-source) [reply README.md:62]: We already did at https://github.com/skydeckai/mcp-rememberizer-vectordb/blob/6f401d036ae1ccfd12a966faf59d41107245c66d/README.md?plain=1#L68C6-L76C12 anh

**Review by oyster — APPROVED** [review]

**Pull request merged**